lib/hyrax/publisher.rb

```diff
@@ -194,5 +194,21 @@ class Publisher
     # @since 3.2.0
     # @macro a_registered_event
     register_event('object.file.uploaded')
+
+    ##
+    # @return Array[Object] the listeners Hyrax subscribes by default.
+    def default_listeners
```

Comment:
How about adding a simple test since this is a public thing which shouldn't go away without red flags?
Maybe something ensuring it returns an array?

Reply:
tests are in. thanks!